azure/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-azureutils",
     "author": "Microsoft Corporation",
-    "version": "2.0.6",
+    "version": "2.1.0",
```

Comment:
Similar to my comment above, since `IResourceGroupWizardContext` is used by so many steps, this will definitely break any extension that updates.